packages/devtools-reps/src/reps/stubs/long-string-with-url.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const urlFullText = "http://example.com?param=" + "a".repeat(2000);
+const urlFullTextLength = urlFullText.length;
+const urlInitialText = urlFullText.substring(0, 1000);
+
+const stubs = new Map();
+stubs.set("testLongUrl", {
```

Comment:
Is there any case in which `fullText` is not accessible? 🤔 If there is, there's currently no way to keep the cropped url's `href` uncropped and valid.

Reply:
yes, if you log the following in the console `"a ".repeat(10000)` , we don't have the fullText in the grip. 
There should be a way then to load the entire text ( see https://bugzilla.mozilla.org/show_bug.cgi?id=1310630).